packages/flutter/lib/src/material/slider.dart

```diff
@@ -182,13 +189,25 @@ class Slider extends StatefulWidget {
        assert(max != null),
        assert(min <= max),
        assert(value >= min && value <= max),
+       assert(secondaryTrackValue == null || (secondaryTrackValue >= min && secondaryTrackValue <= max)),
```

Comment:
Above you included a message with this assert, should you include it here too?  Though maybe it's obvious.

Reply:
done